src/main/webapp/populateData.js

```diff
@@ -340,3 +340,105 @@ function populateGo() {
         }
       });
 }
+
+/**
+ * Populate Plan-mail panel with potential times to read the emails
+ */
+function populatePlanMail() {
+  const planContainer = document.querySelector('#plan');
+  fetch('/plan-mail')
+      .then((response) => {
+        // If response is a 403, user is not authenticated
+        if (response.status === 403) {
+          throw new AuthenticationError();
+        }
+        return response.json();
+      })
+      .then((planMailResponse) => {
+        // Display the potential times to create events
+        const numWordsContainer = document.querySelector('#word-count');
+        numWordsContainer.innerText = planMailResponse.wordCount;
```

Comment:
Suggestion: if you aren't going to use variables like `numWordsContainer` again, then just use the result of the `document.querySelector` inline:
```js
document.querySelector('#word-count').innerText = planMailResponse.wordCount;
```

Reply:
`Resolved`